scripts/remove_past_intervals.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Program:
+#       Remove all pgs' past_intervals for a specified OSD, please use with
+#       caution as there is no turning back.
+# History:
+# 2018/10/13		yanjun			create
```

Comment:
no need to add history in files, git history should be enough

Reply:
Good point.

Reply:
deleted it